src/components/DataGridWithInfiniteScroll/DataGridWithInfiniteScroll.tsx

```diff
@@ -1,6 +1,11 @@
-import React, {useEffect, useState} from "react";
+import React, {useEffect, useRef, useState} from "react";
 import {useFilters, useSortBy, useTable} from "react-table";
+import { Button} from "@dellstorage/clarity-react/forms/button";
+import { Card, CardBlock, CardTitle } from '@dellstorage/clarity-react/cards'
+import { Table } from '@dellstorage/clarity-react/tables'
 import './DatagridCustomStyles.scss';
+import './DatagridCoulumnSelectionStyles.scss';
+import './DatagridDetailPanelStyles.scss';
```

Comment:
can we keep single SCSS file for all datagrid CSS ? import these CSS files in 'DataGridWithInfiniteScroll.SCSS'